updates/update_messages_table.php

```diff
@@ -5,19 +5,30 @@
 
 class UpdateMessagesTable extends Migration
 {
+    const TABLE_NAME = 'rainlab_translate_messages';
+
     public function up()
     {
-        Schema::table('rainlab_translate_messages', function($table)
-        {
-            $table->boolean('found')->default(1);
-        });
+        if (!Schema::hasTable(self::TABLE_NAME)) {
+            return;
+        }
+
+        if (!Schema::hasColumn(self::TABLE_NAME, 'found')) {
+            Schema::table(self::TABLE_NAME, function($table)
+            {
+                $table->boolean('found')->default(1);
+            });
+        }
     }
 
     public function down()
     {
-        Schema::table('rainlab_translate_messages', function($table)
-        {
-            $table->dropColumn('found');
-        });
+        if (!Schema::hasTable(self::TABLE_NAME)) {
+            return;
+        }
+
+        if (Schema::hasColumn(self::TABLE_NAME, 'found')) {
```

Comment:
@mjauvin just as an FYI, I know it doesn't really matter since migration files should probably never be extended, but I would prefer we always use `static::` over `self::` unless there's a really good reason to use `self::`. It's a better practice to keep since that will provide the most flexibility for developers looking to extend the code.